.gitattributes

```diff
@@ -39,6 +41,8 @@ ocamlbuild/TODO          ocaml-typo=missing-header
 ocamldoc/Changes.txt     ocaml-typo=missing-header
 ocamldoc/ocamldoc.sty    ocaml-typo=missing-header
 
+patches/Patches.md       ocaml-typo=very-long-line,long-line,missing-header
```

Comment:
Thanks - it's a feature/bug of GitHub MarkDown it appears! My MarkDown editor uses the GitHub API for rendering, which appears to "respect" all the line breaks in the file, though its internal MarkDown renderer works correctly.
I've updated it to use 80 characters, therefore.